CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [develop]
+
+### Fixed
+- UI Flickering back and forth on live streams after AD's
+
+### Changed
+- Seekbar doesn't use `SmoothPositionUpdater` on Live streams anymore.
```

Comment:
This entry is actually not true 🤔. The `startSmoothPlaybackPositionUpdater` was never used on Live streams. (It was a bug that it was). So please remove this entry.